odl/discr/lp_discr.py

```diff
@@ -785,6 +997,60 @@ def uniform_discr(min_corner, max_corner, nsamples,
                                    **kwargs)
 
 
+def sequence_space(shape, exponent=2.0, impl='numpy', **kwargs):
+    """Return an object mimicing the sequence space ``l^p(R^d)``.
```

Comment:
That's a different story, of course. If we need that at all. Then this is the simplest truncated version of that.